src/rev_ai/apiclient.py

```diff
@@ -361,7 +360,12 @@ def _make_http_request(self, method, url, **kwargs):
             and stream
         :raises: HTTPError
         """
-        response = self.session.request(method, url, **kwargs)
+        headers = self.default_headers.copy()
+        if ('headers' in kwargs):
+            headers.update(kwargs.get('headers'))
+            del kwargs['headers']
```

Comment:
I think this solution is fine. We don't expert to actually modify the headers often, right?

Reply:
The only time's we have to do this is when we're trying to get output, the content type header needs to be added